packages/larva-patterns/components/c-tagline/c-tagline.twig

```diff
@@ -1,5 +1,5 @@
 {% if c_tagline_text %}
-	<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}">{{ c_tagline_text }}</p>
+	<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}" {{ c_tagline_data_attr }}>{{ c_tagline_text }}</p>
```

Comment:
We cannot use this syntax for data attributes due to PHP escaping - 
`<p class="c-tagline {{ modifier_class }} {{ c_tagline_classes }}" {{ c_tagline_data_attr }}>`

It has to be the value of the data attribute. We should avoid having data attributes on objects and components, and handle JS with `.js-*` class selectors.